tools/cldr-apps/WebContent/RefreshRow.jsp

```diff
@@ -1,223 +1,8 @@
-					response.setContentType("application/json");
-					JSONObject dsets = new JSONObject();
-					if(pageId==null) { // requested an xp, not a pageid?
-						for (String n : SortMode.getSortModesFor(xp)) {
-							dsets.put(
-									n,
-									section.createDisplaySet(
-											SortMode.getInstance(n), matcher));
-						}
-						//DataSection.DisplaySet ds = section.getDisplaySet(ctx, matcher);
-						dsets.put("default", SortMode.getSortMode(ctx, section));
-						pageId = section.getPageId();
-					} else {
-					    dsets.put("default",PathHeaderSort.name); // typically PathHeaderSort.name = "ph" 
-					    dsets.put(PathHeaderSort.name,section.createDisplaySet(SortMode.getInstance(PathHeaderSort.name),null)); // the section creates the sort
-					}
-							
-					if(pageId!=null) {
-						if(pageId.getSectionId() == org.unicode.cldr.util.PathHeader.SectionId.Special) {
-	                        JSONWriter r = new JSONWriter(out).object().
-	                                key("err").value("Items not visible - page " + pageId + " section " + pageId.getSectionId()).
-	                                key("err_code").value("E_SPECIAL_SECTION").endObject();
-	                        return;
-						}
-					}
-					
-					try {
-						JSONWriter r = new JSONWriter(out).object()
-								.key("stro").value(STFactory.isReadOnlyLocale(ctx.getLocale()))
-								.key("baseXpath").value(baseXp)
-                                .key("pageId").value((pageId!=null)?pageId.name():null)
-								.key("section").value(section)
-								.key("localeDisplayName").value(ctx.getLocale().getDisplayName())
-								.key("displaySets").value(dsets)
-								.key("dir").value(ctx.getDirectionForLocale())
-								.key("canModify").value(ctx.canModify())
-								.key("locale").value(ctx.getLocale())
-								.key("dataLoadTime").value(et.toString());
-						if(ctx.hasField("dashboard")) {
-							VettingViewerQueue vvq = new VettingViewerQueue();
-							JSONArray issues = VettingViewerQueue.getInstance().getErrorOnPath(ctx.getLocale(), ctx,ctx.session, baseXp);
-							r.key("issues").value(issues);
-						}
-						r.endObject();
-					} catch(Throwable t) {
-						SurveyLog.logException(t, "RefreshRow.jsp write");
-                        JSONWriter r = new JSONWriter(out).object().
-                                key("err").value("Exception on writeSection:"+t.toString()).endObject();
-					}
-					return;
-				}
-				/* Do nothing here if no json parameter (deleted dead code) */
-			}
-			
-	    	} finally {
-	    		// put the name back.
-				curThread.setName(threadName);
-			}
-            %>
\ No newline at end of file
+<%@page import="org.unicode.cldr.web.*"%>
+<%@page language="java" contentType="text/html; charset=UTF-8"
+	import="com.ibm.icu.util.ULocale,org.unicode.cldr.util.*,org.json.*"%>
+<%-- Copyright (C) 2012 IBM and Others. All Rights Reserved --%>
+<%
+	// THIS FILE IS OBSOLETE, to be deleted...
```

Comment:
since it's a JSP, perhaps it should write out an error (not in a comment) just in case someone lands on it accidentally?

Reply:
OK, I'll delete jsp instead.